docsrcs/yui-ibc-solidity/docs/minitoken/interaction.md

```diff
@@ -55,17 +55,30 @@ MiniToken.deployed()
     .then(event => console.log(event));
 ```
 
+## Check the block height on ledger IBC1
+
+When sending a Packet from IBC0 to IBC1, it's necessary to specify the height at which the Packet will timeout.
```

Comment:
@3100  Should we emphasize that the timeout height is obtained from "receiving" chain, not "sending" chain?

Reply:
@bluele Understood. I've made a modification by adding a note that the timeout height is based on the receiving ledger.